mlflow/R/mlflow/R/python.R

```diff
@@ -12,7 +12,11 @@ get_python_bin <- function() {
     stop(paste("MLflow not configured, please run install_mlflow() or ",
                "set MLFLOW_PYTHON_BIN and MLFLOW_BIN environment variables.", sep = ""))
   }
-  file.path(dirname(mlflow_env$python), "python.exe")
+  if (.Platform$OS.type == "windows") {
+    file.path(dirname(mlflow_env$python), "python.exe")
```

Comment:
On windows, what does `mlflow_env$python` look like?

Reply:
"xxx\\xxx\\xxx/r-mlflow-1.25.1/python"; after changing it would become "xxx/xxx/xxx/r-mlflow-1.25.1/python"

Reply:
Can we make `mlflow_env$python` value to be "xxx/xxx/xxx/r-mlflow-1.25.1/python" ?

Reply:
This is the function reticulate:::conda_list returned format, so probably I could raise a PR inside their repo to fix the return format, otherwise I guess we could just fix it here like this?

Reply:
Done.

Reply:
Is `xxx/xxx/xxx/r-mlflow-1.25.1/python` a python exectutable file on windows?

Reply:
It's python.exe

Reply:
To confirm, `mlflow_env$python` looks like `xxx\\xxx\\xxx/r-mlflow-1.25.1/python.exe`, correct?

Reply:
I double checked, it's `xxx\\\xxx\\\xxx\\\r-mlflow-1.25.1/python.exe` and if you take a look at my last change, `mlflow_env$python <- normalizePath(mlflow_env$python, winslash = "/", mustWork = FALSE)` would turn it into `xxx/xxx/xxx/r-mlflow-1.25.1/python.exe`

Reply:
Dose `xxx/xxx/xxx/r-mlflow-1.25.1/python.exe` work on Windows? If it does, it's ok.